packages/dm-core-plugins/src/table/TableRow/TableRow.tsx

```diff
@@ -117,7 +120,7 @@ export function TableRow(props: TTableRow) {
         {config.functionality?.openAsTab && (
           <Table.Cell style={{ textAlign: 'center' }}>
             <Button
-              title="Open"
+              disabled={!item.isSaved}
```

Comment:
UU is not my area of expertize, but shouldn't there be an aria-label here?

Reply:
It has a label/title in the button, so there's no need 👍🏻 For buttons that only have icons and no text for sr to read is where labeling it is more important